src/modules/map/components/LocationPopup.vue

```diff
@@ -126,101 +135,140 @@ export default {
             currentLang: (state) => state.i18n.lang,
             displayLocationPopup: (state) => state.map.displayLocationPopup,
         }),
-        clickCoordinates() {
-            return this.clickInfo?.coordinate
+        coordinate() {
+            return this.clickInfo?.coordinate || null
```

Comment:
I'm not sure if having an optional access here wouldn't return null anyway? (or `undefined` which is also considered `false` in an `if(...)` statement)

Reply:
The optional access did returned `undefined` which caused a warning in the `LocationPopupCopySlot` but changing this component to an optional value solved this issue.